### ISSUE TITLE
Windows binaries 32/64 bit are missing

### ISSUE THREAD

**Issue by ChessGUI** [mp3259/Turochamp#1]
Hello,

can you create Windows binaries 32/64 bit and host these files on your website, please?

**Comment by AnotherLife** [comment]
Yes, is this a working engine? If yes, can you please post a windows build?
